libusb/os/darwin_usb.h

```diff
@@ -120,6 +120,7 @@ struct darwin_cached_device {
   bool                  in_reenumerate;
   int                   capture_count;
   usbi_mutex_t          capture_mutex;
+  usbi_mutex_t          open_mutex;
```

Comment:
What is this mutex protecting? `open_count`? anything else?

Reply:
Just open_count